intel_extension_for_transformers/neural_chat/docker/tgi_serving/tgi.yaml

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the parameter configuration file for NeuralChat Serving.
+
+#################################################################################
+#                             SERVER SETTING                                    #
+#################################################################################
+host: 0.0.0.0
+port: 8000
+
+model_name_or_path: "Intel/neural-chat-7b-v3-1"
+device: "cpu"
+
+serving:
+    framework: "tgi"
+    tgi_engine_params:
+        model_id: "mistralai/Mistral-7B-Instruct-v0.1"
```

Comment:
Why need to set model_id while already configured model_name_or_path?

Reply:
Will integrate into one model_name_or_path.